src/style.css

```diff
@@ -1 +1,93 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.body {
+  margin: 0;
+  padding: 0;
+
+  background-color: rgb(241, 241, 241);
+}
+
+.header {
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+  justify-content: space-between;
+
+  max-width: 1200px;
+  min-width: 1024px;
+
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo__link {
+  display: flex;
+}
+
+.nav {
+  display: flex;
+  align-items: stretch;
+  align-self: stretch;
+  justify-self: flex-end;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__list {
+  display: flex;
+
+  margin: 0;
+  padding: 0;
+
+  list-style-type: none;
+}
+
+.item__link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  text-decoration: none;
+  height: 60px;
```

Comment:
don't hardcode height, use paddings instead

Reply:
Dont know is this is right way to solve this problem, just trying my best
`.item__link {
  display: flex;
  align-items: center;
  position: relative;
  text-decoration: none;
  
  padding: calc(30px - 1em / 2) 0;

  font-size: 12px;
  line-height: 1em;
  font-weight: 500;
  color: #000;
}`

Reply:
Yes, it's correct